### PR TITLE
"It's about time" the post of the talk

### PR DESCRIPTION
**Pull request by batpigandme**
This is just a post-ified version of Davis' talk from rstudio::conf, with the added benefit of a transcript (for those who would rather read), and some more links to clock's documentation.

@DavisVaughan, I'm totally happy to swap you in as author here and change up the language in the intro paragraph so it doesn't sound like you're talking about yourself in the third person. I've written nothing original here, I just didn't want to be presumptuous and start ghost-writing on your account.

### CODE REVIEW

**Review by tracykteal — COMMENTED** [review]
Post is good with me, if it's good with @DavisVaughan!

**Review by DavisVaughan — APPROVED** [review]
Looks great to me as is! Thanks!

File: content/blog/its-about-time/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+At rstudio::conf(2022), Davis Vaughan gave a lightning talk on [clock](https://clock.r-lib.org/), an R package that aims to provide comprehensive and safe handling of date-times. clock goes beyond the date and date-time types that base R provides, implementing new types for year-month, year-quarter, ISO year-week, and many other date-like formats, all with up to nanosecond precision.
```
Comment (hadley) [inline content/blog/its-about-time/index.Rmd:34]: I think we need to say something about lubridate here: i.e. lubridate is not going away, and we expect to update lubridate in the future to use clock as a backend (so clock becomes stringi to lubridate's stringr)

**Review by hadley — APPROVED** [review]

File: content/blog/its-about-time/index.Rmd (hunk 1 above)
Comment (batpigandme) [reply content/blog/its-about-time/index.Rmd]: Done!

**Pull request merged**